src/notebooks/helpers.ts

```diff
@@ -585,6 +585,16 @@ function convertOutputMimeToJupyterOutput(mime: string, value: Uint8Array) {
                     })
                 );
             }
+        } else if (mime.toLowerCase() === 'application/vnd.holoviews_load.v0+json') {
```

Comment:
could this mime type change at some point in the future? The `v0` tells me there might be a `v1` eventually, perhaps 🤔

Reply:
good point